Dockerfile

```diff
@@ -0,0 +1,24 @@
+ARG PYTHON_VERSION=3.10
+FROM python:${PYTHON_VERSION}-slim AS base
+LABEL maintainer="yaroslavbordovoy@gmail.com"
+
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR /app
+
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
+
+COPY . .
```

Comment:
Copying the entire context with `COPY . .` can lead to larger image sizes and potential inclusion of unnecessary files. Ensure that your `.dockerignore` file is properly configured to exclude files that are not needed in the image.